tests/_test_mscolab/test_server.py

```diff
@@ -33,13 +33,19 @@
 from mslib.mscolab.server import check_login, register_user
 from mslib.mscolab.file_manager import FileManager
 from mslib.mscolab.seed import add_user, get_user
+from tests.utils import XML_CONTENT1, XML_CONTENT2
 
 
 class Test_Server:
     @pytest.fixture(autouse=True)
-    def setup(self, mscolab_app):
+    def setup(self, mscolab_app, mscolab_managers):
         self.app = mscolab_app
+        self.sockio, self.cm, self.fm = mscolab_managers
+        self.sm = self.sockio.sm
```

Comment:
There should be no need for `cm`, `fm` and `sm` in this class, to test what it is intended to test.